### ISSUE TITLE
Laravel Livewire: @entangle incorrectly marked as debug statement

### ISSUE THREAD

**Issue by Kovah** [kalessil/phpinspectionsea#1902]
| Subject        | Details                               |
| :------------- | :------------------------------------ |
| Plugin         | Php Inspections (EA Extended)  5.0.0.0|
| Language level | PHP 8.2                               |

Hi, I am not 100'% sure if this really is an issue caused by EA or PhpStorm itself, but the code inspection lists EA as the causing entity.

Current behaviour
--
When using Laravel Livewire inside Blade templates together with Alpine.js, the `@entangle()` directive is marked as a debug statement because EA thinks it is the `print_r()` function. The inspection title is *[EA] Please ensure this is not a forgotten debug statement.*

Code example:

```blade
<div x-data="{ open: @entangle('showDropdown') }">
    <button @click="open = true">Show More...</button>
 
    <ul x-show="open" @click.outside="open = false">
        <li><button wire:click="archive">Archive</button></li>
    </ul>
</div>
```

Code signature of the @entangle directive:
```
public static function entangle($expression): string
```

Actual output of the @entangle directive in the HTML:
```html
<div x-data="{ open: window.Livewire.find('yF8Rm9RZZGOpaifgYhuy').entangle('showDropdown') }">
    <button @click="open = true">Show More...</button>
 
    <ul x-show="open" @click.outside="open = false">
        <li><button wire:click="archive">Archive</button></li>
    </ul>
</div>
```

More details:
- [Livewire code in question](https://github.com/livewire/livewire/blob/2.x/src/LivewireBladeDirectives.php#L14)
- [Livewire docs](https://laravel-livewire.com/docs/2.x/alpine-js#sharing-state)

Expected behaviour
--

PhpSorm does not mark the `@entangle()` function as a debug statement as suggested by the EA plugin. The EA plugin does not think that the function is a `print_r()` statement.

Environment details
---

Information from Help - About dialog
```
PhpStorm 2023.1.2
Build #PS-231.9011.38, built on May 16, 2023
Runtime version: 17.0.6+10-b829.9 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
```

**Comment by ea-inspections-team** [comment]
Thank you for reporting @Kovah!

Based on the inspection of the [source code](https://github.com/kalessil/phpinspectionsea/blob/master/src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/debug/ForgottenDebugOutputInspector.java), I am unsure how `entangle` is being resolved to `print_r`.

It is possible that either PhpStorm or another extension is handling the resolution in this way, but unfortunately, I do not have additional information.
